.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Continuous Integration'
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  check-dist:
+    name: Check dist/ directory
+    uses: actions/reusable-workflows/.github/workflows/check-dist.yml@a8533f184b279cfc1b2dd6a96ed2f097ccf81189
```

Comment:
this commit is quite old, I recommend we bump to the latest here. https://github.com/actions/reusable-workflows/commit/967035ce963867fb956a309c9b67512314bc7c1f

Reply:
This was also recommended below but marked as resolved :/

Reply:
Implemented